### PR TITLE
Add new example never_run tag for precompiled objects

### PR DESCRIPTION
**Pull request by olehborysevych**
Adding new tag for playground examples to upload precompiled output from repo

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by olehborysevych** [comment]
R: @damccorm

File: playground/infrastructure/verify.py [hunk 1]
```diff
@@ -66,7 +72,46 @@ async def _run_and_verify(self, examples: List[Example]):
...
+            Populate fields of the example reading them from the repository.
+            Args:
+                example: beam example that should be verified
+            """
+            path = Path(example.filepath)
+            example_folder = path.parent
+
+            log_file_path = example_folder / self.LOGS_FILENAME
+            # Check if the file exists and read its content
+            if log_file_path.exists():
```
Comment (damccorm) [inline playground/infrastructure/verify.py:99]: If these files don't exist, should we be raising some sort of exception?

**Review by damccorm — COMMENTED** [review]
I'm trying to understand the motivation, could you give me an example of where we would want to use `never_run`?

File: playground/infrastructure/verify.py (hunk 1 above)
Comment (olehborysevych) [reply playground/infrastructure/verify.py:99]: > I'm trying to understand the motivation, could you give me an example of where we would want to use `never_run`?

This is to be able to skip CI and upload logs and output for examples that cannot be run in production. We would like to upload cross-language examples with this tag as we agreed with Kerry.

Comment (olehborysevych) [reply playground/infrastructure/verify.py:99]: > If these files don't exist, should we be raising some sort of exception?
We do not treat this as exception. It's ok if some files are missing.

**Review by damccorm — APPROVED** [review]
Thanks!

**Pull request merged**